src/overlays/actors/ovl_Demo_Tre_Lgt/z_demo_tre_lgt.c

```diff
@@ -147,6 +147,12 @@ s32 DemoTreLgt_PostLimbDraw(GlobalContext* globalCtx, SkelAnimeCurve* skelCurve,
     }
 
     CLOSE_DISPS(globalCtx->state.gfxCtx, "../z_demo_tre_lgt.c", 448);
+
+    //! @bug This function should return a boolean, true if it is to be drawn, but does not match with a return.
+    //! TODO: what happens to v0 here?
+#ifdef AVOID_UB
+    return true;
+#endif
```

Comment:
To answer the todo, #1183 https://github.com/zeldaret/oot/pull/1183/commits/f0201bd5e4fc486d672be645b87577ef245474d1

Reply:
That refers to retail as well, I haven't done any breakpointing in pj64 or anything yet.

Reply:
I checked retail, they all end up putting the opa pointer in v0, so I'll add that in.